src/styles/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  padding: 16px;
+  border: 1px solid transparent;
+}
+
+.card__container {
+  padding-top: 16px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+  /* margin: 32px 19px 40px; */
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: var(--main-color);
+}
+
+.card__product-code {
+  color: var(--second-color);
+  margin-bottom: 16px;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__reviews {
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--main-color);
+}
+
+.card__stars-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__price {
+  line-height: 18px;
+  font-size: 12px;
+  color: var(--second-color);
+}
+
+.card__price-item {
+  color: var(--main-color);
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+}
+
+.card__price-block {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__link {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 16px;
+  width: 166px;
+  height: 40px;
+  font-size: 14px;
+  line-height: 16px;
+  font-weight: 700;
+  color: #fff;
+  background: #00acdc;
+  border-radius: 5px;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.card__link:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;
```

Comment:
checklist point add border to element
![image](https://user-images.githubusercontent.com/70009205/228327760-f9e84278-9cf5-4299-a85a-eead2aef2fca.png)